docker-compose.yaml

```diff
@@ -53,34 +52,47 @@ services:
     ulimits:
       nofile: $NVMEOF_NOFILE
       memlock: -1
-  ceph:
-    image: $QUAY_CEPH:$CEPH_CLUSTER_VERSION
-    container_name: ceph
+  ceph-base:
     build:
       context: .
       dockerfile: Dockerfile.ceph
       args:
         CEPH_CLUSTER_VERSION:
+        CEPH_CLUSTER_CEPH_REPO_BASEURL:
       labels:
         io.ceph.nvmeof:
     environment:
-      CEPH_VSTART_ARGS:
-      VSTART_ARGS: --without-dashboard --memstore
       TOUCHFILE: /tmp/ceph.touch
     entrypoint: >-
-      sh -c './vstart.sh --new $$VSTART_ARGS &&
+      sh -c './vstart.sh --new $$CEPH_VSTART_ARGS &&
       ceph osd pool create rbd &&
+      echo ceph dashboard nvmeof-gateway-add -i <(echo nvmeof-devel:5500) nvmeof.1 &&
```

Comment:
This is currently useless. It should be moved (without the `echo`, of course) into the vstart.sh when invoked with a new flag, like `vstart.sh --nvmf` (note to @nizamial09 ), which would also create the `rbd` pool.